var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -109,23 +171,80 @@ def install(self, spec, prefix):
         options = ["-prefix=%s" % prefix,
                    "-iowrapper",
                    "-pdt=%s" % spec['pdt'].prefix]
-        # If download is active, download and build suggested dependencies
-        if '+download' in spec:
-            options.extend(['-bfd=download',
-                            '-unwind=download',
-                            '-asmdex=download'])
-        else:
-            options.extend(["-bfd=%s" % spec['binutils'].prefix])
-            # TODO : unwind and asmdex are still missing
+#       If download is active, download and build suggested dependencies
+
+        options.extend(["-bfd=%s" % spec['binutils'].prefix])
+        options.extend(["-unwind=%s" % spec['libunwind'].prefix])
 
         if '+scorep' in spec:
             options.append("-scorep=%s" % spec['scorep'].prefix)
 
+        if '+likwid' in spec:
+            options.append("-likwid=%s" % spec['likwid'].prefix)
```

Comment:
@sameershende: question on all of these options: are these disabled by default?

Typically in Spack the policy is to make sure that an explicit argument is provided whether the variant is enabled or disabled, so that the configure system doesn't add `likwid` by default when the variant is set to `~likwid`.  e.g., if this were autotools, we'd be sure to supply either `--with-likwid=<prefix>` or `--without-likwid`.

If TAU has these things disabled by default and that is unlikely to change, then I think this stuff is fine.

Reply:
Yes, these options are disabled by default and that is unlikely to change.